tests/unit/package_managers/test_rpm.py

```diff
@@ -41,6 +41,11 @@
         checksum: sha256:94803b5e1ff601bf4009f223cb53037cdfa2fe559d90251bbe85a3a5bc6d2aab
         size: 14735448
         repoid: updates-source
+    metadata:
```

Comment:
I'm trying to update e2e test with ubi8 image.

Reply:
Added an e2e testcase for module https://github.com/hlin/cachi2-rpm/tree/e2e-module

Reply:
Great, can you please try looking for a module with fewer dependencies to decrease the time needed for the test to run (i.e. decrease the download time), it's not a coincidence I used subversion libs in my Fedora example, because that one had only 4 deps and the packages are just a few KiB each.

Reply:
You can then open a PR against the testing repo to merge the case addition first and then reference it from this PR.

Reply:
Updated https://github.com/hlin/cachi2-rpm/tree/e2e-module to install httpd-tools instead, which only needs to download around 400 KiB in total.
And can you help to create `e2e-module` branch https://github.com/cachito-testing/cachi2-rpm? and then I can create PR with it.

Reply:
> And can you help to create e2e-module branch https://github.com/cachito-testing/cachi2-rpm? and then I can create PR with it.

Sure, I will create it if needed, but hear me out first.
We could further decrease the amount of time needed for the e2e by combining the modularity test with the existing e2e. Not only would it build a single container only, but it would also test the weird corner case scenario (covered by a conventional int. test case already) where you could have multiple "packages" inside a repo, IOW you'd have multiple lockfiles that we'd need to process, one lockfile would contain regular RPM deps and the other would be scoped for modularity.

I'd like to see it consolidated, but admittedly, that's an improvement that can be done as a follow-up, so if you don't feel up for the challenge, let me know and I create a new branch for you in the test repo.

Reply:
> Sure, I will create it if needed, but hear me out first. We could further decrease the amount of time needed for the e2e by combining the modularity test with the existing e2e. Not only would it build a single container only, but it would also test the weird corner case scenario (covered by a conventional int. test case already) where you could have multiple "packages" inside a repo, IOW you'd have multiple lockfiles that we'd need to process, one lockfile would contain regular RPM deps and the other would be scoped for modularity.

I don't understand it clearly that how to build single container with multiple lockfiles

> 
> I'd like to see it consolidated, but admittedly, that's an improvement that can be done as a follow-up, so if you don't feel up for the challenge, let me know and I'll create a new branch for you in the test repo.

Initially, I attempted to update the Containerfile and lockfile directly in the e2e branch. However, as you know, F39+ does not support modularity, which caused significant changes to test_data/rpm_e2e_test. I felt that these changes were not so ideal and then created a new test case with e2e-module branch.  I'd prefer to improve it as a follow-up.

Reply:
Sorry for a long delay in response, I've created this branch for you: https://github.com/cachito-testing/cachi2-rpm/tree/e2e-modularity

Reply:
Updated integration test against https://github.com/cachito-testing/cachi2-rpm/pull/3